tt_llk_blackhole/common/inc/sfpu/ckernel_sfpu_reduce.h

```diff
@@ -223,7 +223,164 @@ inline void calculate_reduce_float()
  */
 constexpr bool is_supported_reduce_format(DataFormat format)
 {
-    return format == DataFormat::Int32 || format == DataFormat::UInt32 || format == DataFormat::Float32;
+    return format == DataFormat::Int32 || format == DataFormat::UInt32 || format == DataFormat::Float32 || format == DataFormat::Float16_b;
+}
+
+//**************************************************************
+// SFPU REDUCE COL IMPLEMENTATION FOR SDPA
```

Comment:
```suggestion
// SFPU REDUCE MAX COL IMPLEMENTATION
```
No need to leak details like this. It's a reduce max column, and it could be used anywhere.